content/en/blog/_posts/2023-05-18-seccomp-profiles-edge.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: blog
+title: "Having fun with seccomp profiles on the edge"
+date: 2023-05-18
+slug: seccomp-profiles-edge
+---
+
+**Author**: Sascha Grunert
+
+The [Security Profiles Operator (SPO)][spo] is a feature-rich
+[operator][operator] for Kubernetes to make managing seccomp, SELinux and
+AppArmor profiles easier than ever. Recording those profiles from scratch is one
+of the key features of this operator, which usually involves the integration
+into large CI/CD systems. Being able to test the recording capabilities of the
+operator in edge cases is one of the recent development efforts of the SPO and
+makes it excitingly easy to play around with seccomp profiles.
+
+[spo]: https://github.com/kubernetes-sigs/security-profiles-operator
+[operator]: https://kubernetes.io/docs/concepts/extend-kubernetes/operator
+
+## Recording seccomp profiles with `spoc record`
+
+The [v0.8.0][spo-latest] release of the Security Profiles Operator shipped a new
+command line interface called `spoc`, a little helper tool for recording and
+replaying seccomp profiles among various other things that are out of scope of
+this blog post.
+
+[spo-latest]: https://github.com/kubernetes-sigs/security-profiles-operator/releases/v0.8.0
+
+Recording a seccomp profile requires a binary to be executed, which can be a
+simple golang application which just calls [`uname(2)`][uname]:
+
+```go
+package main
+
+import (
+	"syscall"
+)
+
+func main() {
+	utsname := syscall.Utsname{}
+	if err := syscall.Uname(&utsname); err != nil {
+		panic(err)
+	}
+}
+```
+
+[uname]: https://man7.org/linux/man-pages/man2/uname.2.html
+
+Building a binary from that code can be done by:
+
+```console
+> go build -o main main.go
+> ldd ./main
+        not a dynamic executable
+```
+
+Now it's possible to download the latest binary of [`spoc` from
+GitHub][spoc-latest] and run the application on Linux with it:
+
+[spoc-latest]: https://github.com/kubernetes-sigs/security-profiles-operator/releases/download/v0.7.1/spoc.amd64
```

Comment:
How come this isn't 0.8.0 as per the text?

Reply:
Yes it should be 0.8.0, too. Fixed.